.circleci/config.yml

```diff
@@ -391,92 +387,101 @@ workflows:
           yarn_run: dev-tariff-duty-calculator-smoketests
           requires:
             - deploy_dev
-          filters:
-            branches:
-              ignore:
-                - /^dependabot/(?!docker/).*/
-                - /^hotfix\/.+/
+          <<: *ignore-main
+
+  Deploy_to_Staging:
+    jobs:
+      - write_docker_tag:
+          name: write_docker_tag_staging
+          environment: staging
+
+      - plan_terraform:
+          name: plan_terraform_staging
+          context: trade-tariff-terraform-aws-staging
+          environment: staging
+          requires:
+            - write_docker_tag_staging
+          <<: *ignore-main
 
       - build:
-          name: build_paas_live
+          name: build_live
           context: trade-tariff
-          filters:
-            branches:
-              only:
-                - main
-                - /^hotfix\/.+/
+          <<: *only-main
+
+      - tariff/build-and-push:
+          name: build_and_push_live
+          context: trade-tariff-terraform-aws-staging
+          environment: staging
+          image_name: tariff-duty-calculator
+          ssm_parameter: "/staging/DUTY_CALCULATOR_ECR_URL"
+          <<: *only-main
+
+      - apply_terraform:
+          name: apply_terraform_staging
+          context: trade-tariff-terraform-aws-staging
+          environment: staging
+          requires:
+            - plan_terraform_staging
+            - build_and_push_live
+          <<: *only-main
 
       - deploy_main_to_staging:
           context: trade-tariff
-          filters:
-            branches:
-              only:
-                - main
-                - /^hotfix\/.+/
           requires:
-            - build_paas_live
+            - build_live
+          <<: *only-main
+
+      - apply_terraform:
+          context: trade-tariff-terraform-aws-staging
+          environment: staging
+          requires:
+            - write_docker_tag_staging
+            - build_and_push_live
+          <<: *only-main
 
       - tariff/smoketests:
-          name: smoketest_staging
           context: trade-tariff
           url: https://staging.trade-tariff.service.gov.uk
           yarn_run: staging-tariff-duty-calculator-smoketests
           requires:
             - deploy_main_to_staging
-          filters:
-            branches:
-              ignore:
-                - /^dependabot/(?!docker/).*/
-                - /^hotfix\/.+/
+          <<: *only-main
 
-      - hold_create_release:
-          type: approval
-          filters:
-            branches:
-              only:
-                - main
-                - /^hotfix\/.+/
+  Deploy-to-Production:
```

Comment:
Not sure what kind of casing this is. Is this basically to differ from non-workflow things?

Reply:
Sorry yeah on review casing is a bit all over the place. If there's a case you prefer I'll redo the whole lot to that? `kebab-case` or `snake_case`?

Reply:
I honestly don't mind too much. Was just surprised :)